variables.tf

```diff
@@ -37,3 +37,24 @@ variable "route_next_hop_type" {
 variable "route_next_hop_in_ip_address" {
   default = "10.10.1.1"
 }
+
+variable "aks_subscription_id" {
+  type        = string
+  description = "The AKS subscription id for the environment.  Set by pipeline."
+}
+
+variable "apim_suffix" {
+  type        = string
+  description = "The suffix to use for the APIM name.  If not set then the environment name will be used."
+  default     = ""
+}
```

Comment:
This variable was needed because the perftest and aat environments use "test" and "stg" in the APIM component names rather than the environment name.  You might be doing a similar translation elsewhere in the module - if so you might be able to consolidate the approaches.